chapter_convolutional-modern/batch-norm.md

```diff
@@ -507,6 +663,13 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 ```
 
+
+
+```{.python .input}
+#@tab tensorflow
+d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
```

Comment:
We haven't defined `train_ch6` for `d2l.tensorflow` yet though. Have you verified that your changes work?

Reply:
actually，in the local environment, I defined a train_ch6 in my package。By the way, I noticed that you also used from d2l import tensflow as d2l, but the package about tensorflow has not been used, are you also defining it locally?

Reply:
It's actually being defined and used in previous chapters. You can find them in d2l/tensorflow.py.